docs/resources/additional-resources.md

```diff
@@ -30,7 +30,27 @@ This page provides a curated list of additional resources for further exploratio
 - [News](https://www.cryptopolitan.com/huawei-introduces-defactor-to-the-world/)
 - [E-Book](https://www.defactor.com/resources/lending-pools-unpacked)
 
-## Video Tutorials
-
+## Toolkit Test Environment
+### Urls
+- [Assets Test Environment](https://assets.defactor.dev/)
+- [Pools Test Environment](https://assets.defactor.dev/)
```

Comment:
@yannickeosams  the correct link is https://pools.defactor.dev

Reply:
we will also soon have a test environment for engage https://engage.defactor.dev